src/Channel/__tests__/Home.spec.jsx

```diff
@@ -0,0 +1,123 @@
+import { render, screen, fireEvent, waitFor } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import axios from "axios";
+import { MemoryRouter } from "react-router-dom";
+import { describe, test, expect, beforeEach, vi } from "vitest";
+
+import Home from "@/Channel/components/Home";
+import useCategorizeChannels from "@/Channel/hooks/useCategorizeChannels";
+
+vi.mock("axios");
+vi.mock("@/Channel/hooks/useChannels", () => ({
+  default: vi.fn(),
+}));
+vi.mock("@/Channel/hooks/useCategorizeChannels", () => {
+  const mockedCategorize = vi.fn();
+  return {
+    default: mockedCategorize,
+    __esModule: true,
+  };
+});
+vi.mock("@/User/hooks/useUserData", () => ({
+  default: vi.fn(),
+}));
+vi.mock("@/User/hooks/useUserId", () => ({
+  default: vi.fn(),
+}));
+vi.mock("@/Channel/hooks/useChannelNavigation", () => ({
+  default: () => mockNavigate,
+}));
+vi.mock("@/Channel/hooks/useToggleFavorite", () => ({
+  default: () => mockToggleFavorite,
+}));
```

Comment:
```suggestion
vi.mock("axios");

vi.mock("@/Channel/hooks/useChannels", () => ({
  default: vi.fn(),
}));

vi.mock("@/Channel/hooks/useCategorizeChannels", () => {
  const mockedCategorize = vi.fn();
  return {
    default: mockedCategorize,
    __esModule: true,
  };
});

vi.mock("@/User/hooks/useUserData", () => ({
  default: vi.fn(),
}));

vi.mock("@/User/hooks/useUserId", () => ({
  default: vi.fn(),
}));

vi.mock("@/Channel/hooks/useChannelNavigation", () => ({
  default: () => mockNavigate,
}));

vi.mock("@/Channel/hooks/useToggleFavorite", () => ({
  default: () => mockToggleFavorite,
}));
```
테스트 코드를 잘 작성해주셨네요!
다만, [예시](https://github.com/charmverse/app.charmverse.io/blob/8e47480a3fffaad4034ad4425d3447e02c1ddeeb/vite.setup.ts#L16)처럼 `vi.mock` 구문 간 개행을 적용하면 전체적으로 포맷이 통일되고 가독성도 더 좋아지는 것 같습니다!

Reply:
희주님의 제안대로 `vi.mock` 구문 간 개행이 가독성 측면에서 확실히 더 좋은 것 같네요. 좋은 예시까지 공유해주셔서 감사드립니다. 해당 사항 반영하겠습니다~